### PR TITLE
Genesis fix for x86_64

### PR DESCRIPTION
**Pull request by abhishek-sa1**
### The PR is to fix issue

#7401 
#7385

### CODE REVIEW

**Comment by abhishek-sa1** [comment]
@ocfmatt @Obihoernchen

**Comment by abhishek-sa1** [comment]
New genesis rpm and deb file has to be created in xCAT tarball.

**Review by ocfmatt — COMMENTED** [review]
Hello,

I've had reports of similar issues with drivers from OCF engineers when using Intel E810 family nics so I am definitely in favour of this change to alleviate that.

What is the purpose of the raw ipmi command on doxcat line 427 and 434?

Regards,
Matt.

**Comment by abhishek-sa1** [comment]
@ocfmatt Sometime even after apply `ipmitool chassis bootdev pxe`, it doesn't get effective due to timeout of boot. By setting `ipmitool raw 0x00 0x08 0x03 0x08` ignores the timeout and apply PXE in all the cases.

**Comment by ocfmatt** [comment]
> @ocfmatt Sometime even after apply `ipmitool chassis bootdev pxe`, it doesn't get effective due to timeout of boot(i.e. it should boot within 60 seconds). `ipmitool raw 0x00 0x08 0x03 0x08` is valid Bootflag clearing ipmi command and apply PXE in all the cases. Verified in Dell PowerEdge servers.

@Obihoernchen I haven't seen this on any recent OCF deployments. Have you run into this issue before?

File: xCAT-genesis-scripts/usr/bin/doxcat [hunk 1]
```diff
@@ -424,12 +424,14 @@ while :; do
 		if [ $IPMI_SUPPORT -ne 0 ]; then
                     # Set boot from network will cause OpenPOWER server wait at petitboot menu, so do nothing here
                     if uname -m | grep x86_64; then
+				ipmitool raw 0x00 0x08 0x03 0x08
```
Comment (samveen) [inline xCAT-genesis-scripts/usr/bin/doxcat:427]: 4 spaces per level instead of tabs

**Comment by abhishek-sa1** [comment]
> > @ocfmatt Sometime even after apply `ipmitool chassis bootdev pxe`, it doesn't get effective due to timeout of boot(i.e. it should boot within 60 seconds). `ipmitool raw 0x00 0x08 0x03 0x08` is valid Bootflag clearing ipmi command and apply PXE in all the cases. Verified in Dell PowerEdge servers.
> 
> @Obihoernchen I haven't seen this on any recent OCF deployments. Have you run into this issue before?

@ocfmatt  Can you verify the change in your environment.  I can revert this change if required.

**Comment by ocfmatt** [comment]
> > > @ocfmatt Sometime even after apply `ipmitool chassis bootdev pxe`, it doesn't get effective due to timeout of boot(i.e. it should boot within 60 seconds). `ipmitool raw 0x00 0x08 0x03 0x08` is valid Bootflag clearing ipmi command and apply PXE in all the cases. Verified in Dell PowerEdge servers.
> > 
> > 
> > @Obihoernchen I haven't seen this on any recent OCF deployments. Have you run into this issue before?
> 
> @ocfmatt Can you verify the change in your environment. I can revert this change if required.

I will test this and see what impact it has on non-Dell hardware. Worst case scenario I will cherry pick everything but the raw IPMI from your PR.

Thanks,
Matt.

**Comment by abhishek-sa1** [comment]
@ocfmatt @Obihoernchen Is it verified in non-dell hardware?

**Comment by rshawocf** [comment]
@abhishek-sa1 I am testing this and will work with @ocfmatt on it. So far I have replicated the original problem https://github.com/xcat2/xcat-core/issues/7385 and going to test the fix

**Comment by rshawocf** [comment]
Update - in our R&D environment the code in PR7460 works fine on Virtual Machines. We still need to test this on non-Dell hardware and will look to do this as soon as possible,

**Review by Obihoernchen — APPROVED** [review]

**Comment by ocfmatt** [comment]
This has already been merged but I have been able to confirm these changes do not have any detrimental impact to a Lenovo SR630 V2 server.

**Pull request merged**